From 7bae91fd94b6bcbcc43270a86f09b3d55147efcf Mon Sep 17 00:00:00 2001
From: Eduardo Ferreira <eduardo.barbosa@toradex.com>
Date: Mon, 29 Jan 2024 10:30:18 -0300
Subject: [PATCH] systemd-networkd-wait-online.service.in: use --any by default

Use --any by default when waiting for a network interface to be fully
configured, otherwise it blocks until all the available interfaces are
in the configured state.

Obs.: Inherited from LmP.

Upstream-Status: Inappropriate [TorizonCore specific]

Signed-off-by: Sergio Prado <sergio.prado@toradex.com>
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Eduardo Ferreira <eduardo.barbosa@toradex.com>
---
 units/systemd-networkd-wait-online.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-networkd-wait-online.service.in b/units/systemd-networkd-wait-online.service.in
index 7768121f5f..7d26e04fe3 100644
--- a/units/systemd-networkd-wait-online.service.in
+++ b/units/systemd-networkd-wait-online.service.in
@@ -19,7 +19,7 @@ Before=network-online.target shutdown.target
 
 [Service]
 Type=oneshot
-ExecStart={{LIBEXECDIR}}/systemd-networkd-wait-online
+ExecStart={{LIBEXECDIR}}/systemd-networkd-wait-online --any
 RemainAfterExit=yes
 
 [Install]
-- 
2.34.1

